website/package.json

```diff
@@ -100,7 +100,7 @@
     "json-to-ast": "^2.1.0",
     "lodash.isequal": "^4.5.0",
     "luaparse": "^0.2.1",
-    "meriyah": "latest",
+    "meriyah": "^1.6.5",
```

Comment:
@aladdin-add v. 1.6.7 is the latest version

Reply:
updated the lockfile to use latest -- v1.6.8